apps/fluent-tester/src/testPages.ts

```diff
@@ -117,7 +117,7 @@ export const tests: TestDescription[] = [
     platforms: ['ios'],
   },
   {
-    name: 'Divider',
+    name: 'Divider V1',
```

Comment:
nit:  we need to mark this as V1 - I am of understanding that are adding V1 for components that have a non-v1 variant existing.

Reply:
My understanding was different , thanks for the clarification. Changed it.